src/components/tasks/card/index.tsx

```diff
@@ -57,17 +67,17 @@ const Card: FC<Props> = ({ content }) => {
             width={iconWidth}
             height={iconHeight}
           />
-          <span className={classNames.cardSpecialFont}>
-            Due Date
-          </span>
+          <span className={classNames.cardSpecialFont}>Due Date</span>
           <span className={classNames.cardStrongFont}>{fromNowEndsOn}</span>
         </span>
       </div>
       <div className={classNames.cardItems}>
         <span className={classNames.cardSpecialFont}>
           Started
           {' '}
-          {fromNowStartedOn}
+          {
+            fromNowStartedOn
+          }
```

Comment:
Can we revert this? not a required change I guess.

Reply:
it showed me a red squiggly line saying that we can only have one jsx expression per line. I'll change it

Reply:
reverted. Please check.